lib/internal/streams/readable.js

```diff
@@ -1371,6 +1371,18 @@ Readable.from = function(iterable, opts) {
   return from(Readable, iterable, opts);
 };
 
+async function * streamSeriesIterator(streams) {
+  for await (const stream of streams) {
+    for await (const chunk of stream) {
+      yield chunk;
+    }
+  }
+}
+
+Readable.concat = function(streams, opts) {
```

Comment:
To make `Readable.concat.length` equals to `1`:
```suggestion
Readable.concat = function(streams, opts = undefined) {
```